### PR TITLE
Task/tlt 1870/course management dashboard

### PR DESCRIPTION
**Pull request by sapnamysore**
This PR contains changes for migrating manage people and manage section into the Canvas Course Admin tools with the UI  changes specified here https://jira.huit.harvard.edu/browse/TLT-1870

[Most of the code has been through PR in chunks.]

### CODE REVIEW

File: canvas_course_admin_tools/settings/base.py [hunk 1]
```diff
@@ -27,27 +27,30 @@
...
+    'manage_people',
+    'manage_sections',
 )
 
 MIDDLEWARE_CLASSES = (
     'djangular.middleware.DjangularUrlMiddleware',
     'django.contrib.sessions.middleware.SessionMiddleware',
     'django.middleware.common.CommonMiddleware',
-    'django.middleware.csrf.CsrfViewMiddleware',
+    # 'django.middleware.csrf.CsrfViewMiddleware',
```
Comment (sapnamysore) [inline canvas_course_admin_tools/settings/base.py]: We should delete this

**Comment by sapnamysore** [comment]
I looked thru the tons of files. We should spot check the app  when we deploy this to dev.
Looks good! 👍

File: canvas_course_admin_tools/settings/base.py (hunk 1 above)
Comment (elliottyates) [inline canvas_course_admin_tools/settings/base.py]: This reminds me that it was on my to-do list to figure out how to integrate this with our apps, plus the `XFrameOptionsMiddleware`. But that can be tech debt / research.

File: canvas_course_admin_tools/requirements/local.txt [hunk 2]
```diff
@@ -4,14 +4,22 @@
...
+flake8==2.5.4
+mock==2.0.0
+oauthlib==1.1.1
+pep8==1.7.0
+pyvirtualdisplay==0.2
+requests-oauthlib==0.6.1
+selenium==2.53.2
+xlrd==0.9.4
 
 # icommons_common unit tests depend on ccsw, and ccsw depends on icommons_common.
```
Comment (elliottyates) [inline canvas_course_admin_tools/requirements/local.txt:19]: If we're not actually going to run icommons_common unit tests, perhaps we can remove this completely and note it in our tech debt ticket.

File: canvas_course_admin_tools/settings/base.py [hunk 3]
```diff
@@ -268,5 +282,40 @@
...
+# lti_permissions.LtiPermission permission field for each app in the project
+# that uses it (this is used on the dashboard to determine which app links to
+# make available to the user
+CUSTOM_LTI_PERMISSIONS = {
+    'isites_migration': 'im_import_files',
+    'manage_people': 'manage_people',
+}
+
+MANAGE_PEOPLE_MSGS = {
+    'lti_request': 'There was a problem fulfilling your request. Please contact HUIT support.',
```
Comment (elliottyates) [inline canvas_course_admin_tools/settings/base.py]: I'd love to see these two in a single high-level/app-level dict called MANAGE_PEOPLE, but no biggie.

**Comment by elliottyates** [comment]
Looks good! I'd like to add back in the oauth tests -- will PR that now. Then we can merge it!

File: canvas_course_admin_tools/requirements/local.txt (hunk 2 above)
Comment (elliottyates) [inline canvas_course_admin_tools/requirements/local.txt:19]: Actually, looks like they run ok now! :)

**Comment by elliottyates** [comment]
Waiting on #41.

**Pull request merged**